### ISSUE TITLE
comsat-jersey-grizzly

### ISSUE THREAD

**Issue by icode** [puniverse/comsat#88]
has grizzly server for jersey?

**Comment by circlespainter** [comment]
[I read](http://www.oracle.com/webfolder/technetwork/tutorials/obe/java/basic_grizzly_jersey/jersey-grizzly-json-service.html) [Jersey](https://jersey.java.net/) can run on [Grizzly](https://grizzly.java.net/) but I don't think Grizzly offers a servlets container personality and `comsat-jersey-server` builds upon `comsat-servlet`, so I think at present you can't have a fiber-based JAX-RS server on top of Grizzly.

A Grizzly-Quasar integration should be possible to write as well as a Grizzly-Jersey-Quasar one on top but I have no idea of the effort and of the demand; if it's important to you just try and tell how it goes, then some help could arrive from us and other interested people in the community.

Another option could be using [Glassfish](https://glassfish.java.net/) which has a Grizzly container and being a compliant servlets implementation it could (should?) work with `comsat-jersey-server` but it's untested.

**Comment by icode** [comment]
@circlespainter no, grizzly + jersey not use servlet, is a lightweight service. like netty

**Comment by icode** [comment]
@circlespainter or `jersey-server` unwith `servlet`, `jersey` not must need `servlet`
